sparse_strips/vello_common/src/coarse.rs

```diff
@@ -308,12 +308,12 @@ impl Wide {
         } else {
             // Calculate the y range from first to last strip in wide tile coordinates
             let wtile_y0 = strips[0].strip_y();
-            let wtile_y1 = strips[n_strips - 1].strip_y() + 1;
+            let wtile_y1 = strips[n_strips.saturating_sub(1)].strip_y() + 1;
```

Comment:
It is worth noting `saturating_sub` doesn't come completely free, but it's not worth worrying about that too much here as this path isn't too hot.

x86: https://godbolt.org/z/qP6r85q7f
AArch64: https://godbolt.org/z/dh91xGbbG